msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/IAuthenticationResult.java

```diff
@@ -44,4 +44,9 @@ public interface IAuthenticationResult extends Serializable {
      * @return access token expiration date
      */
     java.util.Date expiresOnDate();
+
+    /**
+     * @return whether this is a proof-of-possession token
+     */
+    Boolean isPopAuthorization();
```

Comment:
1. Adding a member to an interface is a breaking change (since you'll break ppl using this interface). Can you use default methods instead?
2. .NET uses "string TokenType" instead to differentiate "Bearer", "pop" and potentially others (SSH-cert is the other one).